apps/api/src/domain/models/audio-item/commands/translate-audio-item-name/translate-audio-item-name.command.ts

```diff
@@ -0,0 +1,31 @@
+import { ICommandBase, LanguageCode } from '@coscrad/api-interfaces';
+import { Command } from '@coscrad/commands';
+import { NestedDataType, NonEmptyString } from '@coscrad/data-types';
+import { LanguageCodeEnum } from '../../../../../domain/common/entities/multilingual-text';
+import { AudioItemCompositeIdentifier } from '../../entities/audio-item-composite-identifier';
+import { TRANSLATE_AUDIO_ITEM_NAME } from '../constants';
+
+@Command({
+    type: TRANSLATE_AUDIO_ITEM_NAME,
+    label: 'Translate Audio Item Name',
+    description: 'Translate the name of a audio item to another language',
+})
+export class TranslateAudioItemName implements ICommandBase {
+    @NestedDataType(AudioItemCompositeIdentifier, {
+        label: 'Composite Identifier',
+        description: 'system-wide unique identifier',
+    })
+    readonly aggregateCompositeIdentifier: AudioItemCompositeIdentifier;
+
+    @LanguageCodeEnum({
+        label: 'language code',
+        description: 'the language in which you are naming the new audio item name',
```

Comment:
the language in which you are translating the audio item's name

Reply:
done